src/SeedAddress.svelte

```diff
@@ -1,16 +1,14 @@
 <script lang="ts">
-  import type { Config } from "./config";
   import { formatSeedAddress, formatSeedId, toClipboard } from "./utils";
 
-  export let config: Config;
   export let id: string;
   export let host: string;
   export let port: number;
 
   let seedCopied = false;
 
   const copySeed = (seedId: string, seedHost: string) => {
-    return () => toClipboard(formatSeedAddress(seedId, seedHost, config)).then(() => {
+    return () => toClipboard(formatSeedAddress(seedId, seedHost, port)).then(() => {
```

Comment:
We we're passing the whole`config` object into this really simple component, only to show the port number, so I took the liberty to simplify it by only accepting the port number and updating the `formatSeedAddress` to accept a number instead of `config`.